app/globals.css

```diff
@@ -0,0 +1,46 @@
+.marker {
+  /* Pin */
+  width: 26.2px;
+  height: 41px;
+  background-image: url("/marker.svg");
+  background-size: cover;
+  border-radius: 0;
+}
+
+.soft-story {
```

Comment:
the soft story CSS should be replaced with Mapbox Layer styling in an upcoming PR (since we need to replace soft story Markers with Layers)